metadata-ingestion/tests/integration/git/test_git_clone.py

```diff
@@ -83,11 +84,37 @@ def test_github_branch():
     assert config.branch_for_clone == "main"
 
 
+def test_url_subdir() -> None:
+    git_ref = GitReference(repo="https://github.com/org/repo", url_subdir="dbt")
+    assert (
+        git_ref.get_url_for_file_path("model.sql")
+        == "https://github.com/org/repo/blob/main/dbt/model.sql"
+    )
+
+    git_ref = GitReference(repo="https://gitlab.com/org/repo", url_subdir="dbt")
+    assert (
+        git_ref.get_url_for_file_path("model.sql")
+        == "https://gitlab.com/org/repo/-/blob/main/dbt/model.sql"
```

Comment:
I'm not familiar with gitlab and this `-` in the middle called my attention. 

Apparently, the dashless variant also works https://gitlab.com/gitlab-org/gitlab/-/issues/365984

Reply:
huh interesting

The `/-/` stuff was already in there, and given that it still works, I don't think it makes sense to change right now